src/java/htsjdk/variant/variantcontext/VariantContext.java

```diff
@@ -1605,14 +1606,33 @@ private final Genotype fullyDecodeGenotypes(final Genotype g, final VCFHeader he
     // tribble integration routines -- not for public consumption
     //
     // ---------------------------------------------------------------------------------------------------------
+    @Deprecated
     public String getChr() {
+        return getContig();
+    }
+
+    @Override
+    public String getContig() {
         return contig;
     }
 
+    /**
+     * @return 1-based inclusive start position of the Variant,
+     * INDEL events usually start on the first unaltered reference base before the INDEL
+     * @warning be aware that the start position of the VariantContext is defined in terms of the start position specified in the
+     * underlying vcf file, VariantContexts representing the same biological event may have different start positions depending on the
+     * specifics of the vcf file they are derived from
+     */
     public int getStart() {
         return (int)start;
     }
 
+    /**
+     * @return 1-based closed end position of the Variant, if the END info field is specified that value is returned, otherwise the end is
+     * the start + reference allele length - 1
+     * insertions end on the reference base before the insertion event
+     * deletions end on the final deleted reference base
+     */
```

Comment:
The formatting of this comment is confusing (no periods, newlines, capitalization).